packages/block-editor/src/components/block-has-dot/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { Fill } from '@wordpress/components';
+
+function BlockHasDot( {
```

Comment:
It'd be good to give this a more specific name like `BlockUnsavedChangesIndicators`.

Reply:
Yes, this makes the most sense.